src/cloudai/workloads/megatron_bridge/slurm_command_gen_strategy.py

```
@@ -166,12 +292,22 @@ def _normalize_cuda_graph_scope_arg(self, val: Any) -> str:
         parts = [p.strip().strip("\"'") for p in s.split(",") if p.strip()]
         return ",".join(parts)
 
-    def _wrap_launcher_for_job_id_and_quiet_output(self, launcher_cmd: str, launcher_python: str) -> str:
+    def _wrap_launcher_for_job_id_and_quiet_output(
+        self,
+        launcher_cmd: str,
+        launcher_python: str,
+        pre_hook_sbatch_path: Optional[Path] = None,
+        base_slurm_params: str = "",
+        capture_nodelist: bool = False,
```

Ruff (0.15.20)

[warning] Boolean-typed positional argument in function definition

(FBT001)